web/pandas_web.py

```diff
@@ -430,8 +430,8 @@ def main(
                 content = extend_base_template(body, context["main"]["base_template"])
             context["base_url"] = "".join(["../"] * os.path.normpath(fname).count("/"))
             content = jinja_env.from_string(content).render(**context)
-            fname = os.path.splitext(fname)[0] + ".html"
-            with open(os.path.join(target_path, fname), "w") as f:
+            fname_splitted = os.path.splitext(fname)[0] + ".html"
+            with open(os.path.join(target_path, fname_splitted), "w") as f:
```

Comment:
I think  this is just about replacing the extension with `html`, so maybe `fname_html` would be better?

Reply:
Agree, just updated name.